chapters/operatorsandexpressions.tex

```diff
@@ -803,6 +804,20 @@ \subsection{Derivative and Special Purpose Operators with Function Syntax}\label
 Since it is difficult to prove whether expressions are continuous, tools can dynamically check the continuity such that \lstinline!x2! only generates an error when the integration reaches the discontinuity.
 \end{example}
 
+\begin{example}
+Exception for reinit:
+\begin{lstlisting}[language=modelica]
+  Real x4, x5, x6;
+equation
+  when sample(1, 1) then
+    reinit(x4, 0);
+  end when;
+  der(x4) = time;
```

Comment:
Is that allowed? It would be nice to add a comment about it here too.

Reply:
I'm unsure what the question specifically refers to, but the entire example is intended to be legal.